src/storage/examples/src/objects/print_file_acl.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START storage_print_file_acl]
+use google_cloud_storage::client::StorageControl;
+
+pub async fn sample(client: &StorageControl, bucket_id: &str) -> anyhow::Result<()> {
+    const NAME: &str = "object-with-acl";
```

Comment:
I think the builds will fail because this object doesn't exist. `"object-to-read"` would work.

FYI, here is where we create the objects referenced in the samples:

https://github.com/googleapis/google-cloud-rust/blob/0e326e639a0299bc6cd213a3c127d01cd7094507/src/storage/examples/src/lib.rs#L389

Reply:
ah, that was a miss on my part.